test/geometry/adjoints.jl

```diff
@@ -107,10 +107,10 @@ end
         for ind in ((2.5, 2.5), (5, 5))
             if t <: Colorant
                 zy = Zygote.gradient((x,y)->_sep(ImageTransformations._getindex(x,y)), itp, ind)
-                @test zy[2] ≈ fieldsum.(Interpolations.gradient(itp, ind...))
+                @test zy[2] == Tuple(_sep.(Interpolations.gradient(itp, ind...)))
             else
                 zy = Zygote.gradient((x,y)->ImageTransformations._getindex(x,y), itp, ind)
-                @test zy[2] ≈ Interpolations.gradient(itp, ind...)
+                @test zy[2] == Tuple(Interpolations.gradient(itp, ind...))
```

Comment:
Why is the test changed to `==`? It should be testing for approximation.

Reply:
The main reason for using  ==  over the approximation was that tests was failing after using the ≈ symbol, so I figured out maybe approximation can't be used to compare tuples, hence I went with a more direct straight forward == .  Tried to google it but wasn't able to find anything significant. That was the problem that I was facing, hence I went with == symbol. If any changes can be made here would love to change it.

Reply:
Have made the changes to use approximation rather than == by creating a custom function as approx (  ≈) cannot be used to compare tuples and will lead to test errors